package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "gitlab-easy-time-tracking",
   "version": "0.0.1",
+  "private": true,
```

Comment:
What does this mean?

Reply:
From npm documentation:

> If you set "private": true in your package.json, then npm will refuse to publish it.
> This is a way to prevent accidental publication of private repositories.